sirepo/job_supervisor.py

```diff
@@ -145,14 +145,14 @@ def init():
     _DB_DIR = sirepo.srdb.root().join(_DB_SUBDIR)
     cfg = pkconfig.init(
         job_cache_secs=(300, int, 'when to re-read job state from disk'),
-        max_hours=dict(
-            analysis=(.04, float, 'maximum run-time for analysis job',),
```

Comment:
does the change from float affect anything? I don't think so, just asking.

Reply:
Not really but it is a breaking change if anyone was using floats in their config.

Reply:
The question is: does changing from float to int (which was committed) break anything? I assume you hit all cases so probably not.